packages/core/src/state-management/__tests__/cache-eviction.test.ts

```diff
@@ -106,6 +109,7 @@ test('RunningState stops updating after evicted', async () => {
   expect(runningState1.isStopped).toBeTruthy(); // RunningState is stopped after eviction
 
   const stream2 = await createDocument();
```

Comment:
This test too I feel like we could make unique documents and just use explicit document loads, rather than deterministic creates, without affecting what the test is actually trying to test

Reply:
Actually it looks like this is probably the case for all the tests in this file.  Nothing about this test is explicitly about testing the behavior of deterministic document lookups.  I'm pretty sure they were just written this way because it was convenient and easy at the time. Now I think we'd be better served by explicit load operations rather than jumping through hoops to keep using deterministic documents when nothing about the core of the test needs that